src/Behat/Behat/Output/Statistics/PhaseStatistics.php

```diff
@@ -111,7 +111,7 @@ public function registerHookStat(HookStat $stat)
     /**
      * Returns counters for different scenario result codes.
      *
-     * @return array<TestResult::*, int>
+     * @return array<StepResult::*, int>
```

Comment:
This includes one case that is defined in StepResult, so we use the derived class